app/src/test/java/com/concordia/mcga/factories/FloorFactoryTest.java

```diff
@@ -0,0 +1,146 @@
+package com.concordia.mcga.factories;
+
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import com.concordia.mcga.exceptions.MCGADatabaseException;
+import com.concordia.mcga.models.Building;
+import com.concordia.mcga.utilities.pathfinding.IndoorMapTile;
+import com.concordia.mcga.utilities.pathfinding.TiledMap;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.MarkerOptions;
+import junit.framework.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class FloorFactoryTest {
```

Comment:
Can be refactored but I think it's unnecessary as unit test code does not have to be perfect.